src/util.cc

```diff
@@ -538,25 +550,29 @@ std::string StringToHex(const std::string &input) {
   return output;
 }
 
+constexpr std::uint64_t ExpTo1024(size_t n) {
+  return static_cast<std::uint64_t>(1) << (10 * n);
+}
+
 void BytesToHuman(char *buf, size_t size, uint64_t n) {
   double d;
 
-  if (n < 1024) {
+  if (n < ExpTo1024(1)) {
```

Comment:
actually this code is copied from redis.
but for this change, i don't feel it is clear, maybe we can define `KB MB GB TB PB`.

And this change is not related to theme.

Reply:
It just replace `1024*1024*...*1024` to something like `1024^n`, so I don't think it will affect readability. 

> And this change is not related to theme
Agree. Maybe move it to a separate PR.

Reply:
These changes are reverted, I'll prepare a seperate PR to illustrate them.